openstack/compute/v2/extensions/migrate/requests.go

```diff
@@ -9,3 +9,43 @@ func Migrate(client *gophercloud.ServiceClient, id string) (r MigrateResult) {
 	_, r.Err = client.Post(actionURL(client, id), map[string]interface{}{"migrate": nil}, nil, nil)
 	return
 }
+
+// LiveMigrateOptsBuilder allows extensions to add additional parameters to the
+// LiveMigrate request.
+type LiveMigrateOptsBuilder interface {
+	ToLiveMigrateMap() (map[string]interface{}, error)
+}
+
+// LiveMigrateOpts specifies parameters of live migrate action.
+type LiveMigrateOpts struct {
+
+	// The host to which to migrate the server.
+	// If this parameter is None, the scheduler chooses a host.
+	Host *string `json:"host"`
```

Comment:
Is this so you can specify `""` as a valid host value?

Reply:
@jtopjian It will be "valid", but I reckon that "host" has to be always in the request body according to [this line](https://github.com/openstack/nova/blob/master/nova/api/openstack/compute/migrate_server.py#L82).
Moreover there's no clear mention as for [migrate-server-migrate-action](https://developer.openstack.org/api-ref/compute/#migrate-server-migrate-action)

> If you specify null or **don’t specify this parameter,** the scheduler chooses a host.

I might be wrong please correct me.